package.json

```diff
@@ -126,5 +125,6 @@
   "pre-commit": [
     "lint"
   ],
-  "private": true
+  "private": true,
+  "dependencies": {}
```

Comment:
On purpose?

Reply:
no, `npm uninstall ember-click-outside` bolted this one. remove?

Reply:
removed it.

Reply:
Yea, it's done that to me, maybe just leave it this time.

Reply:
or not 😁